Lib/lighthouse/errors.hpp

```diff
@@ -0,0 +1,20 @@
+//
+//  errors.hpp
+//  Lighthouse Camera
+//
+//  Created by David Teller on 31/01/17.
+//  Copyright © 2017 Lighthouse. All rights reserved.
+//
+
+#ifndef lighthouse_errors_h
+#define lighthouse_errors_h
+
+enum LighthouseError {
+  // The operation could not complete because of an error in video.{cpp, hpp} // FIXME: Make it more detailed.
+  ERROR_COULD_NOT_CAPTURE = 0,
```

Comment:
Initiallly, I went with a `class enum`, which makes this redundant, but they apparently don't work in Obj-C (at least in the bridge). So yes, probably a good idea.